### PR TITLE
Update CP I2C Scan Example

### PR DESCRIPTION
**Pull request by caternuson**
An attempt to update code example to show creating I2C bus manually. QT Py 2040 and Pi Pico are two good examples where existing code does not work and lot's of users end up running into this.

### CODE REVIEW

**Review by ladyada — APPROVED** [review]

File: CircuitPython_Templates/i2c_scan/code.py [hunk 1]
```diff
@@ -1,15 +1,29 @@
 """CircuitPython I2C Device Address Scan"""
+# If you run this and it seems to hang, try manually unlocking
+# your I2C bus from the REPL with
+#  >>> import board
+#  >>> board.I2C().unlock()
+
```
Comment (kattni) [inline CircuitPython_Templates/i2c_scan/code.py:6]: Please remove this, as it is explained in the guide page associated with this example.

**Review by TheKitty — APPROVED** [review]

**Review by kattni — CHANGES_REQUESTED** [review]
Looks good to me. One thing below.

File: CircuitPython_Templates/i2c_scan/code.py (hunk 1 above)
Comment (caternuson) [reply CircuitPython_Templates/i2c_scan/code.py:6]: cool. thanks. i wanted to do that anyway. wasn't sure.

File: CircuitPython_Essentials/CircuitPython_I2C_Scan/code.py [hunk 2]
```diff
@@ -1,9 +1,4 @@
 """CircuitPython I2C Device Address Scan"""
-# If you run this and it seems to hang, try manually unlocking
-# your I2C bus from the REPL with
-#  >>> import board
-#  >>> board.I2C().unlock()
-
```
Comment (kattni) [inline CircuitPython_Essentials/CircuitPython_I2C_Scan/code.py:6]: Apologies for not being clear. Please leave it in this copy of the code, it is not explained in the Essentials guide, only in the template page.

**Review by kattni — CHANGES_REQUESTED** [review]
One last thing, sorry!

**Review by kattni — APPROVED** [review]

**Comment by kattni** [comment]
@caternuson Thanks for adding this!

**Pull request merged**